extensions/binding/org.eclipse.smarthome.binding.tradfri/ESH-INF/config/config.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:config-description="http://eclipse.org/smarthome/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/config-description/v1.0.0 http://eclipse.org/smarthome/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="thing-type:tradfri:device">
+		<parameter-group name="device">
```

Comment:
Please don't use parameter groups if there is just a single parameter to put into it...

Reply:
No problem. I removed it.